LICENSE

```diff
@@ -1,4 +1,4 @@
-                                 Apache License
+                                  Apache License
```

Comment:
I would squash this commit with the previous one.

Reply:
I purposefully made it a different commit to separate the original ported files from new additions by me.